### PR TITLE
Add Taiwan phone format

### PR DESCRIPTION
**Pull request by Supaplextw**

### CODE REVIEW

File: data/phone-formats.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
         "ru": "+7 495 1234567",
         "kz": "+7 495 1234567",
         "vn": "+84 1 234 5678",
-        "hk": "+852 12345678"
+        "hk": "+852 12345678",
+        "tw": "+886 2 2720 8889",
```
Comment (bhousel) [inline data/phone-formats.json]: I get a parse error because of this trailing comma, can you remove it?

**Comment by bhousel** [comment]
Is "+886 2 2720 8889" an example number or somebody's actual phone number?  I'd expect an example number to look like "+886 123456789".  This format looks a bit different from [what Wikipedia suggests](https://en.wikipedia.org/wiki/Telephone_numbers_in_Taiwan).

**Comment by Supaplextw** [comment]
Sure, I'll fix it.

**Comment by bhousel** [comment]
Thanks! Merged as:

6ed2dce Use fake phone number for placeholder
8f40627 fix format: remove comma
ec61344 Add Taiwan phone format

**Comment by Supaplextw** [comment]
Wow, I didn't notice it use fake number, I've just pick the phone number from Taipei City Government. Thanks for transformation of fake number.

**Pull request closed without merging**